offscreen/scripts/offscreen.ts

```diff
@@ -17,3 +18,5 @@ const parentStream = new BrowserRuntimePostMessageStream({
 parentStream.on('data', (data) => {
   console.log('Offscreen Document received data from service worker', data);
```

Comment:
This line should probably be removed in this PR because the stream now receives messages from snaps! However, its helpful to keep right up until we have validated the approach.